src/loader.js

```diff
@@ -0,0 +1,38 @@
+let isLoadingLibrary = false
+const callbacks = []
+
+function hasLoadedLibrary() {
+  return window.google && window.google.maps && window.google.maps.places
+}
+
+export function loadGooglePlacesLibrary(apiKey, callback) {
```

Comment:
might consider a  default arg here instead of the `callback &&` below

Reply:
Here's some discussion of ways to provide a default value for a callback parameter, though the accepted answer has been downvoted more than it's been upvoted: https://stackoverflow.com/questions/40890252/es6-default-value-parameter-for-callback

Would this be preferable (since it's more readable than the `callback && ...` approach)?
```javascript
if (callback) {
  callbacks.push(callback)
}
```